locals.tf

```diff
@@ -3,4 +3,20 @@ locals {
     scope      = "terraform-managed"
     repository = "jenkins-infra/azure-net"
   }
+
+  vpn = {
+    shorthostname = "vpn-test"
+    username      = "jenkins-infra-team"
+    allowed_ips = {
+      dduportal   = "109.88.253.68/32"
+      lemeurherve = "176.185.227.180/32"
+      smerle33    = "82.64.5.129/32"
+      olblak      = "109.128.249.199/32"
+      olblak_bis  = "86.130.79.46/32"
```

Comment:
```suggestion
```

Let's only keep active admins IPs. Olivier is still admin in Azure so he can totally add his IP if really needed. WDYT?

Reply:
> Olivier is still admin in Azure so he can totally add his IP if really needed. 

I think modifications of these resources via the Azure Portal should be avoided, and would be overwritten anyway.